src/Components/FeelingRoom/FeelingRoom.js

```diff
@@ -156,16 +162,75 @@ class FeelingRoom extends Component {
       return this.createFineRoom()
     }
   }
+
   startFeeling = () => {
     this.setState({isFeeling: true})
   }
-  playSound(e) {
-    let audio = e.target.parentNode.parentNode.firstChild
-    audio.play()
+
+  playSound = (e) => {
+    if(!this.state.hasAudio){
+      alert('Select a kind of audio')
+    } else if (e.target.id == 'svg'){
+      let audio = e.target.parentNode.parentNode.firstChild
```

Comment:
This is handled very interestingly. I would love to go over the inner workings of this as a group to better understand how implementing audio is working. Not a priority 1, but definitely would like to think about how to refactor

Reply:
Same!
Don't love all of the ifs but am not sure a better way to deal with them at this point....